model-inference/decisionTree/experiments/convert_trained_model_to_framework.py

```diff
@@ -126,11 +139,11 @@ def convert_to_onnx_model(model, config):
 def convert_to_treelite_model(model, config):
     #converting to TreeLite model
     #Prerequisite: install treelite (https://treelite.readthedocs.io/en/latest/install.html)
-    if MODEL == "xgboost":
+    if MODEL in {"xgboost", "lightgbm"}:
         treelite_time_start = time.time()
         model_path = relative2abspath("models", f"{DATASET}_{MODEL}_{config['num_trees']}_{config['depth']}.model")
-        model.save_model(model_path)
-        treelite_model = treelite.Model.load(model_path, model_format='xgboost')
+        model.save_model(model_path)  # TODO: Why are we saving this directly to a .model format?
```

Comment:
To distinguish xgboost and lightgbm, we can name them with different file extension

Reply:
I saw that Treelite Models have a .model extension. I understand how this is working, and will remove this TODO.